src/platform/plugins/shared/workflows_management/common/schema.ts

```diff
@@ -99,7 +99,7 @@ const AlertSchema = z.object({
 
 const SummarizedAlertsChunkSchema = z.object({
   count: z.number(),
-  data: z.array(AlertSchema),
+  data: z.array(z.union([AlertSchema, z.any()])),
```

Comment:
this is for https://github.com/elastic/security-team/issues/13781 right? why do we need it btw?